aten/src/ATen/native/ConvUtils.h

```diff
@@ -5,6 +5,76 @@
 
 namespace at { namespace native {
 
+// NOLINTNEXTLINE(cppcoreguidelines-pro-type-member-init)
+struct ConvParams {
+  std::vector<int64_t> stride;
+  std::vector<int64_t> padding;
+  std::vector<int64_t> dilation;
+  bool transposed;
+  std::vector<int64_t> output_padding;
+  int groups;
```

Comment:
nit: You don't need to fix this because this is a move, but this looks like a bug: groups seems like it should be int64_t?

Reply:
> > Some backends aren't tested yet:
> > miopen (for AMD ROCm)
> > nnpack (for mobile)
> > xnnpack (for mobile)
> > winograd 3x3 (for mobile)
> 
> I just started reading through this, but what is the plan for those? Is it fine to ignore them in this PR or do they need to be handled here?

Current plan is to ignore them for this PR but add a note that they should be handled at some point. I was able to get miopen tested, so that just leaves the mobile backends. I can open a mobile issue indicating that they're not tested and leave it to those folks to deal with it.

Reply:
Yeah I agree with you - seems like a bug. Think it's worth opening an issue for?